### ISSUE TITLE
The accuracy of 3D reconstruction

### ISSUE THREAD

**Issue by danglu999** [zju3dv/NeuralRecon#50]
Hi,
Thanks a lot for your great work!
I used your pre-trained weights and replaced the content of the folder (sampleData) for 3D reconstruction, while there are some objects that are not recognized as 3D objects and not successfully reconstructed. 

The following pictures illustrate the situation.
**Generated mesh**
<img width="843" alt="Screen Shot 2021-09-07 at 15 44 21" src="https://user-images.githubusercontent.com/69308651/132306346-40162683-7cf0-40f6-a588-c3af83d420cb.png">

**Real situation**
<img width="1042" alt="real world" src="https://user-images.githubusercontent.com/69308651/132306369-7a6fd1c0-cb19-407f-a1ff-b0d9ce7d0ebe.png">

I wonder that would the results be much better if i firstly train the mode on full ScanNet dataset?
and would you please let know how the results can be improved?

Looking forward to your reply and thanks!

**Comment by JiamingSuen** [comment]
> I wonder that would the results be much better if i firstly train the mode on full ScanNet dataset?
Yes training on the full ScanNet dataset would potentially improve the accuracy.

> and would you please let know how the results can be improved?
Please make sure that the camera is moving slowly and have enough views with good co-visibility. Other than better data capturing techniques, there is no obvious trick to improve the accuracy, or otherwise we will include it into the paper😆.

**Comment by HeCraneChen** [comment]
> > I wonder that would the results be much better if i firstly train the mode on full ScanNet dataset?
> 
> Yes training on the full ScanNet dataset would potentially improve the accuracy.
> 
> > and would you please let know how the results can be improved?
> 
> Please make sure that the camera is moving slowly and have enough views with good co-visibility. Other than better data capturing techniques, there is no obvious trick to improve the accuracy, or otherwise we will include it into the paper😆.

If the user has moved the camera relatively fast. Is there any chance to compensate the performance by adjusting the parameters? For example, instead of considering 9 views each time, could we consider more views? Do you think that might help at all?

**Comment by JiamingSuen** [comment]
> If the user has moved the camera relatively fast. Is there any chance to compensate the performance by adjusting the parameters? For example, instead of considering 9 views each time, could we consider more views? Do you think that might help at all?

Yes, I believe it will definitely help by using more keyframes when the camera is moving fast.
